core/imageroot/var/lib/nethserver/node/actions/remove-module/50update

```diff
@@ -48,6 +48,9 @@ else # Rootless module
     if [[ ${preserve} == "yes" ]]; then
         userdel "${mid}"
     else
+        # Avoid /etc/passwd locking by erasing home directory separately:
+        # it is a long operation!
+        eval rm -rf ~"${mid}"
```

Comment:
Why use `eval`?

Reply:
Without it does not work. See tilde expansion: https://www.gnu.org/software/bash/manual/html_node/Tilde-Expansion.html

Reply:
That's eval is a little bit scary, maybe it's better to use `getent passwd <user>"?